routes/events.py

```diff
@@ -70,7 +70,7 @@ async def get_event_by_status(event_id):
     tags=["Events"],
     status_code=201,
 )
-async def events_by_location(lat: str, lon: str, radius: int = 10):
+async def events_by_location(lat: float, lon: float, radius: int = 10):
 
     if not lat or not lon:
```

Comment:
extremely unimportant comment that can totally be disregarded and is borderline unnecessary but this could be simplified into a more readable:

`if not (lan and lon):`

(no hurt feelings if not refactored :)

Reply:
Genius move, will implement ASAP